libs/platform/unit/platform_unit_test.cpp

```diff
@@ -221,32 +221,43 @@ TEST_CASE("trampoline_test", "[platform]")
 }
 
 static ebpf_helper_function_prototype_t _helper_functions[] = {
-    {1,
-     "bpf_map_lookup_elem",
-     EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
-    {2,
-     "bpf_map_update_elem",
-     EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE}},
-    {3,
-     "bpf_map_delete_elem",
-     EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
+    {
+        1,
+        "bpf_map_lookup_elem",
+        EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+        {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY},
+        0,
+    },
+    {
+        2,
+        "bpf_map_update_elem",
+        EBPF_RETURN_TYPE_INTEGER,
+        {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE},
+        0,
+    },
+    {
+        3,
+        "bpf_map_delete_elem",
+        EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+        {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY},
+        0,
+    },
     {4,
```

Comment:
nit: be consistent.  238-239 have a newline but here there is no newline before the number